2-mandatory/1-weather-report.js

```diff
@@ -12,9 +12,18 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
-}
+    let temperatureReportResults = [ ];
+
+    for (i = 0; i < cities.length; i++) {
```

Comment:
Suggestion: Use a `for (city of cities)` loop here. This reduces the amount of code you need to write and it is easier for a reader to understand your intention.